### PR TITLE
feat: water body api (LAN-764)

### PR DESCRIPTION
**Pull request by barredterra**
- feat: water body api (LAN-764)
- docs: organization and water body API (LAN-764)
- Import water bodies from GeoJSON file

Todo:

- [x] add managing organization for each water body to the API response
- [x] ensure map is read-only for role "LANDA Member"

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+### Create Demo Accounts
+
+This app provide a bench command `make-demo-accounts`. This is useful for testing permissions. You can use it to create a **Member**, **Member Function** and **User** for every existing **Member Function Category**, in an existing local organization.
```
Comment (marination) [inline docs/cli.md]:
```suggestion
This app provides a bench command `make-demo-accounts`. This is useful for testing permissions. You can use it to create a **Member**, **Member Function** and **User** for every existing **Member Function Category**, in an existing local organization.
```

File: docs/cli.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+Now you can run the following command (replace `$SITE` and `$ORGANIZATION` with your specific values):
+
+```bash
+bench --site $SITE make-demo-accounts $ORGANIZATION
+```
+
+### Import water bodies from GeoJSON
+
+This app provide a bench command `import-geojson`. This is useful for importing water body shapes from a GeoJSON file.
```
Comment (marination) [inline docs/cli.md]:
```suggestion
This app provides a bench command `import-geojson`. This is useful for importing water body shapes from a GeoJSON file.
```

File: landa/commands/__init__.py [hunk 3]
```diff
@@ -95,6 +98,50 @@ def update_organization_series(context, dry_run=False):
...
+		water_body_id = feature.get("properties", {}).get("GEW_ID")
+		if not water_body_id:
+			continue
+
+		if not frappe.db.exists("Water Body", water_body_id):
+			not_found.append(water_body_id)
+			continue
+
+		feature["properties"].pop("GEW_ID")
+		feature["properties"].pop("GEW_NA")
```
Comment (marination) [inline landa/commands/__init__.py]: Import breaks if there is no `GEW_NA` key. Is this key mandatory like the `GEW_ID` key. If so it must be mentioned in the docs or else handled.
```py
 File ".../apps/landa/landa/commands/__init__.py", line 132, in import_features
    feature["properties"].pop("GEW_NA")
KeyError: 'GEW_NA'
```

**Review by marination — CHANGES_REQUESTED** [review]

**Comment by marination** [comment]
Tested the rest. LGTM after changes are resolved.

Screenshot 2023-06-06 at 2 26 40 PM

_imported from geojson file_

**Review by marination — DISMISSED** [review]

**Comment by marination** [comment]
@barredterra good to merge?

**Comment by barredterra** [comment]
@marination there is still an outstanding feature (custom markers) that was supposed to be part of this PR. Let's keep this open for now. I'll check with the customer if we should merge this as a first step.

**Pull request merged**